misc/snapshotter/Dockerfile

```diff
@@ -3,20 +3,33 @@ FROM ubuntu:20.04 AS sourcer
 RUN apt update; apt install --no-install-recommends -y curl wget ca-certificates
 RUN export NYDUS_VERSION=$(curl --silent "https://api.github.com/repos/dragonflyoss/image-service/releases/latest" | grep -Po '"tag_name": "\K.*?(?=")'); \
     wget https://github.com/dragonflyoss/image-service/releases/download/$NYDUS_VERSION/nydus-static-$NYDUS_VERSION-linux-amd64.tgz; \
+    echo $NYDUS_VERSION > /.nydus_version; \
     tar xzf nydus-static-$NYDUS_VERSION-linux-amd64.tgz
 RUN mv nydus-static/* /
 
 FROM ubuntu:20.04
 
 WORKDIR /root/
-RUN mkdir -p /usr/local/bin/ /etc/nydus/ /var/lib/nydus/cache/ /tmp/blobs/
-COPY --from=sourcer /nydusd /nydus-image /usr/local/bin/
-COPY containerd-nydus-grpc /usr/local/bin/
-COPY nydusd-config.fusedev.json /etc/nydus/config.json
-COPY nydusd-config-localfs.json /etc/nydus/localfs.json
+
 RUN apt update && \
     apt install --no-install-recommends -y ca-certificates && \
     rm -rf /var/cache/apt/* /var/lib/apt/lists/*
+
+VOLUME /var/lib/containerd-nydus /run/containerd-nydus
+
+COPY --from=sourcer /.nydus_version /.nydus_version
+
+## FIXME: how to pass nydus snapshotter version?
+## use: build-args?
+ENV NYDUS_SNAPSHOTTER_VERSION=vx.y.z
+
+RUN mkdir -p /usr/local/bin/ /etc/nydus/ /var/lib/containerd-nydus/cache /tmp/blobs/
+COPY --from=sourcer /nydus* /usr/local/bin/
+# FIXME: need an universal method to get containerd-nydus-grpc binary
+COPY containerd-nydus-grpc /usr/local/bin/
```

Comment:
Does `containerd-nydus-grpc` exist in `misc/snapshotter` ?

Reply:
No, that is why I added a FIXME.
The `containerd-nydus-grpc` binary is download by [the `download artifacts` step](https://github.com/containerd/nydus-snapshotter/pull/238/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R88-R92).

Indeed this line is unchanged. IMO we can improve it by add an `image` target in the Makefile to help users to not copy the binary manually.

Reply:
SGTM